docker/Dockerfile.scratch

```diff
@@ -37,10 +37,11 @@ COPY --from=stage --chown=otel:otel /etc/otel /etc/otel
 COPY --from=stage --chown=otel:otel /etc/otel/storage /etc/otel/storage
 
 COPY observiq-otel-collector /collector/observiq-otel-collector
+COPY opampsupervisor /collector/opampsupervisor
 
 ENV OIQ_OTEL_COLLECTOR_HOME=/etc/otel
 ENV OIQ_OTEL_COLLECTOR_STORAGE=/etc/otel/storage
 
 USER otel
 WORKDIR /etc/otel
-ENTRYPOINT ["/collector/observiq-otel-collector"]
+ENTRYPOINT [ "/collector/opampsupervisor" ]
```

Comment:
This is unrelated to this PR, but something we should consider.

@BinaryFissionGames is it possible to configure the supervisor to exit if the collector sub process crashes? I know on a typical host (Linux / Windows) we want it to restart the process, but with containers it would be great if the container died.

Reply:
There's no option for that currently. I think we'd need to discuss upstream how that'd act. Definitely an issue we'll need to figure out.